packages/api/get-service-by-id.ts

```diff
@@ -56,20 +56,20 @@ export const main = async (event: APIGatewayEvent): Promise<APIGatewayProxyResul
 export const getQueryInput = (event: APIGatewayEvent): ServiceByIdQueryInput => {
     const { pathParameters } = event;
 
-    const nocCode = pathParameters?.["nocCode"] ?? "";
+    const nocCode = pathParameters?.nocCode ?? "";
 
     if (!nocCode) {
         throw new ClientError("NOC must be provided");
     }
 
-    const serviceId = pathParameters?.["serviceId"] ?? "";
+    const serviceId = pathParameters?.serviceId;
```

Comment:
would this not need ?? "" at the end?

Reply:
Since we're checking for the existence of serviceId on the next line there's no real benefit to defaulting to an empty string here